serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc

```diff
@@ -1,54 +1,107 @@
-= Workflow Definitions in Kogito Serverless Workflow Tools extension
+= Workflow Definitions
 :compat-mode!:
 // Metadata:
-:description: Workflow Definitions in Kogito Serverless Workflow Tools extension
+:description: Workflow Definitions
 :keywords: kogito, workflow, serverless, Quarkus, Dev UI, definitions
 
-In Kogito Serverless Workflow Tools extension, the Workflow Definitions page displays the list of available workflow definitions, which are used to trigger the cloud events and start the workflow instances. Also, the Workflow Definitions page enables you to add filters to the workflow definitions list.
+In Kogito Serverless Workflow Tools extension, the *Workflow Definitions* page displays the list of available workflows in
+the Kogito project and helps you to start new workflow instances either via REST or by triggering Cloud Events.
 
 .Workflow Definitions page
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-workflow-definitions.png[]
 
-The table on the Workflow Definitions page displays the following details:
+In *Workflow Definitions* table you'll find the list of workflows that shows the following details:
 
 * *Workflow Name*: Displays the name of a workflow definition.
 * *Endpoint*: Displays the URL to a workflow definition.
-* *Actions*: Provides a button to start a new workflow.
+* *Actions*: Provides the *Start new Workflow* button to start workflow instances via REST.
 
-The filters on the Workflow Definitions page enables you to add filters to the table. To search for a specific workflow definition, enter the name of the workflow in the *Filter by workflow name* field and click *Apply Filter* button. The matching workflow definitions appear as a chip below the search field. 
+In the table toolbar provides the ability to apply filters to the table or to trigger cloud events that can start workflow instances
 
-To clear the applied filters, you can click the *Reset to default* button. Also, to fetch newly added workflow definitions, click on the refresh icon next to the *Apply Filter* button.
+== Applying filters to the Workflow Definitions table
```

Comment:
Is this a procedure module? If yes, then we can convert the following sentences into numbered steps.